sdk/azcore/policy_anonymous_credential.go

```diff
@@ -6,13 +6,19 @@
 
```

Comment:
Is it expected customers will use this?  If not we should place in the azcore/policy/... package

Reply:
Yes, anonymous credential is first-class for SDKs that support SAS authorization.